dbt/profiles.yml

```diff
@@ -1,5 +1,5 @@
 dbt-civis:
-  target: "{{ env_var('DBT_TARGET') }}"
+  target: "{{ env_var('DBT_TARGET', 'redshift_adapter') }}"
```

Comment:
Nit: I think we could get rid of this line entirely, since `DBT_TARGET` is an environment variable which dbt understands and it'll fall back to the `default` target which is defined immediately below if the env var isn't present.